### PR TITLE
fix: ec2 key pairs tagging

### PR DESCRIPTION
**Pull request by MacLikorne**

### CODE REVIEW

File: lib/aws/bootstrap/ec2-ssh-keys.j2.tf [hunk 1]
```diff
@@ -3,5 +3,5 @@ resource "aws_key_pair" "qovery_ssh_key_{{ kubernetes_cluster_id }}" {
   public_key = var.ec2_ssh_default_key.public_key
 
   # Don't add tags, there is an issue, the key will disappear, making the autoscaler stuck because of this (in the best case)
```
Comment (deimosfr) [inline lib/aws/bootstrap/ec2-ssh-keys.j2.tf:5]: no need this comment anymore

File: lib/aws/bootstrap/eks-master-cluster.j2.tf [hunk 2]
```diff
@@ -9,6 +9,7 @@ locals {
     ClusterId = var.kubernetes_cluster_id,
     ClusterName = var.kubernetes_cluster_name,
     Region = var.region
+    CreationDate = var.kubernetes_cluster_creation_date
```
Comment (deimosfr) [inline lib/aws/bootstrap/eks-master-cluster.j2.tf:12]: can't we just add it where it's necessary?

File: src/cloud_provider/aws/kubernetes/mod.rs [hunk 3]
```diff
@@ -320,6 +321,7 @@ impl<'a> EKS<'a> {
         // AWS - EKS
         context.insert("eks_cidr_subnet", &eks_cidr_subnet.clone());
         context.insert("kubernetes_cluster_name", &self.name());
+        context.insert("kubernetes_cluster_creation_date", &Utc::now().format("%Y-%m-%d %H:%M:%S").to_string());
```
Comment (deimosfr) [inline src/cloud_provider/aws/kubernetes/mod.rs:324]: It won't work as the date will be reset anytime the engine will be triggered

**Comment by deimosfr** [comment]
Please wait tests to pass before merging it

File: lib/aws/bootstrap/eks-master-cluster.j2.tf [hunk 4]
```diff
@@ -6,13 +6,18 @@ locals {
 
 locals {
   tags_eks = {
-    ClusterId = var.kubernetes_cluster_id,
+    ClusterId = var.kubernetes_cluster_id
     ClusterName = var.kubernetes_cluster_name,
     Region = var.region
+    creationDate = time_static.on_cluster_create.rfc3339
```
Comment (deimosfr) [inline lib/aws/bootstrap/eks-master-cluster.j2.tf:12]: Are you sure it works? Doesn't it change the date on re-apply?

Comment (MacLikorne) [reply lib/aws/bootstrap/eks-master-cluster.j2.tf:12]: Tried it, doesn't change things after first apply.

**Pull request merged**